docs/about/Removed.rst

```diff
@@ -20,7 +20,7 @@ Replaced by `gui/launcher --minimal <gui/launcher>`.
 
 create-items
 ============
-Replaced by `gui/create-item --multi`.
+Replaced by ``gui/create-item --multi``.
```

Comment:
Would be nice to keep the link working here, so:

```suggestion
Replaced by `gui/create-item --multi <gui/create-item>`.
```

Reply:
why is this changelog change even in this change? it doesn't seem related

Reply:
I'm also a little confused here.